docs/HowTo/Introduction.md

```diff
@@ -0,0 +1,118 @@
+(there is a [work item](https://github.com/Microsoft/dotnet-apiport/issues/117) to convert this to a DNX application).
+
+The tool understands three commands:
+
+### Analyze
+
+Arguably the most important function of the tool is its ability to analyze an assembly. This can take in a file, collection of
+files, or a directory of assemblies.  
+
+`ApiPort.exe analyze <options>`
+
+The current options are:
+
+```
+ -f, --file=VALUE           [Required] Path to assembly file or directory of
+                               assemblies.
+  -o, --out=VALUE            Output file name
+  -d, --description=VALUE    Description of the submission
+  -t, --target=VALUE         The target you want to check against.
+  -r, --resultFormat=VALUE   The report output format
+  -p, --showNonPortableApis  Calculate non-portable APIs
+  -b, --showBreakingChanges  Calculate breaking changes on full .NET Framework
+      --noDefaultIgnoreFile  Do not use the standard assembly ignore list
+                               when analyzing breaking changes. The default
+                               ignore list can be found at KnownSafeBreaks.json
+  -i, --ignoreAssemblyFile=VALUE
+                             Specifies a json file defining assemblies that
+                               should not be analyzed for specific targets
+                               while analyzing breaking changes. This can be
+                               useful for excluding assemblies that are known
+                               to not regress on certain .NET Framework
+                               versions due to breaking changes. Note that,
+                               currently, this parameter only affects breaking
+                               change analysis; not portability analysis.
+  -s, --suppressBreakingChange=VALUE
+                             Specifies a breaking change (by ID) to suppress
+                               during breaking change analysis. Any breaking
+                               changes with IDs specified for suppression will
+                               not be reported.
+  -h, -?, --help             Show help
+```
+
+For more details on analyze for portability look [here](PlatformPortability.md) and for breaking change analysis, please
+look [here](BreakingChanges.md).
+
+### List targets
+
+The targets available to analyze against are retrieved from a service that is updated regularly (except when running in
+[offline mode](OfflineMode.mds)).  These are ever growing, and include the previous versions of the framework, although
+it defaults to the most current platforms and versions.
+
+For example, the command `ApiPort.exe listTargets` will output:
+
+```
+Available Targets:
+- .NET Native [Version: 1.0]
+- ASP.NET 5 [Version: 1.0*]
+- Windows [Version: 8.0; 8.1*]
+- .NET Framework [Version: 1.1; 2.0; 3.0; 3.5; 4.0; 4.5; 4.5.1; 4.5.2; 4.6*]
+- Mono [Version: 4.5]
```

Comment:
Thanks @akoeplinger! I added an issue so it won't get lost again: #144. Can you put some info on that issue as to how to determine which version of Mono we have (we have a drop of the assemblies that are indexed for the service)?